packages/framework/assets/js/admin/components/order.js

```diff
@@ -46,7 +49,10 @@ export default class OrderPreview {
                         });
                     }
                 },
-                out: function () {}
+                out: function () {
+                    _this.$previewBox.parents('table').removeClass('overflow-visible').addClass('overflow-hidden');
+                    _this.$previewBox.parents('table').parent().removeClass('overflow-visible').addClass('overflow-hidden');
```

Comment:
as above

Reply:
And next point: classes `overflow-visible` and `overflow-hidden` are used in this place for four times. I preffer create constants with these class names.

Reply:
Added to constants.